Source/WebCore/dom/Element.cpp

```diff
@@ -1975,6 +1976,43 @@ const AtomString& Element::getAttributeNS(const AtomString& namespaceURI, const
     return getAttribute(QualifiedName(nullAtom(), localName, namespaceURI));
 }
 
+static ExceptionOr<String> trustedTypesCompliantAttributeValue(const String& attributeType, const TrustedTypeOrString& value, Element* element, String sink)
```

Comment:
Change this to take the TrustedType enum instead of a String

Reply:
As discussed, we leave it as String for now and do the clean up at a later stage.